Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -405,6 +406,8 @@
     /* Flag for checking of Strong matching is waiting on GAID fetch */
     private boolean performCookieBasedStrongMatchingOnGAIDAvailable = false;
 
+    public static final String SDK_VERSION = "2.8.0";
```

Comment:
Good point. Actually we  have  this defined in couple of places. We can define one in gradle and use the same anywhere.